code/lib/preview-api/template/stories/rendering.stories.ts

```diff
@@ -66,3 +66,31 @@ export const ChangeArgs = {
     await expect(button).toHaveFocus();
   },
 };
+
+let loadedLabel = 'Initial';
+
+/**
+ * This story demonstrates what happens when rendering (loaders) have side effects, and can possibly interleave with each other
+ * Triggering multiple force remounts quickly should only result in a single remount in the end
+ * and the label should be 'Loaded. Click Me' at the end. If loaders are interleaving it would result in a label of 'Error: Interleaved loaders. Click Me'
+ * Similarly, changing args rapidly should only cause one rerender at a time, producing the same result.
+ */
+export const SlowLoader = {
```

Comment:
Shall I disable this one in chromatic, as it is e2e tested and hard to judge if it regressed in chromatic?

Reply:
Sounds good to me.